json-guides/bulkhead.json

```diff
@@ -165,14 +165,66 @@
             ]
         },
         {
-            "name": "AddingAsyncAnnotation",
-            "title": "Adding the @Asynchronous annotation",
+            "name": "AsyncBulkheadAnnotation",
+            "title": "Adding the @Asynchronous annotation with Bulkhead",
             "description": [  
-                "Thread pool style Bulkhead"        
+                "When <code>@Bulkhead</code> is used with <code>@Asynchronous</code>, the thread pool isolation approach will be used. The thread pool approach allows to configure the maximum concurrent requests together with the waiting queue size. When a request cannot be added to the waiting queue, BulkheadException will be thrown.</br>",
+                "Multiple virtual financial advisors can service the requests at the same time. If there are no financial advisor available, requests are allow in the waiting queue. When the maximum number of requests on the waiting queue are reach, an error message will be display.",
+                "You can modify these two parameters:",
```

Comment:
This step is on top of the semaphore step. I don't think you should tell them to modify two parameters. You only need to introduce waitingTaskQueue parameter here.